_docs/management/space-egress.md

```diff
@@ -26,45 +26,40 @@ A summary of each of the ASGs that can be applied to your space are as follows:
 
 When you push your application to cloud.gov, the staging process may require outbound connections to the public internet to fetch dependencies and software modules. As such, during the staging process, your app will run under the `public-egress` until it is staged and ready to run. Once this process is complete, your app will run under the ASGs that have been applied, either by default or by modifications that have been made to your space.
 
 For applications that need access to S3, you have the option of running them under the `public-egress` ASG, or running them in the `restricted-egress` ASG, and using a proxy application (e.g., [squid proxy](http://www.squid-cache.org/), [HA proxy](http://www.haproxy.org/), etc.) to proxy traffic to your S3 bucket(s). Reference implementations showing how to do this will be available soon, or you may reach out to the cloud.gov team for support.
 
-## Managing egress settings for your org or space
-
-To inspect or modify the ASG that apply to your spaces, you can use the following `cf` CLI subcommands. Run `cf <subcommand> --help` to find out more about a specific command, or consult [the cf CLI documentation site](https://cli.cloudfoundry.org/en-US/v6/).
+To inspect or modify the ASG that apply to your spaces, you can use the following cf CLI subcommands. Run `cf <subcommand> --help` to find out more about a specific command, or consult [the cf CLI documentation site](https://cli.cloudfoundry.org/en-US/v6/).
 
 | CF CLI subcommand | Description | 
 | :- | :- |
 | `cf security-group`                         | Show a single security group |
 | `cf security-groups`                        | List all security groups |
 | `cf bind-security-group`                    | Bind a security group to a particular space, or all existing spaces of an org |
-| `cf unbind-security-group`                  | Unbind a security group from a space |
+| `cf unbind-security-group`                 | Unbind a security group from a space |
+| `cf bind-staging-security-group`            | Bind a security group to the list of security groups to be used for staging applications globally |
 | `cf staging-security-groups`                | List security groups globally configured for staging applications |
+| `cf unbind-staging-security-group`          | Unbind a security group from the set of security groups for staging applications globally |
+| `cf bind-running-security-group`            | Bind a security group to the list of security groups to be used for running applications |
 | `cf running-security-groups`                | List security groups globally configured for running applications |
-
-To adjust the security groups for a particular space, you must have the [SpaceManager role](https://docs.cloudfoundry.org/concepts/roles.html#roles). (To check the roles for a given space, use the `cf space-users` command.)
```

Comment:
Why were these docs removed?

Reply:
Yeah thats weird. I dont think I did that, or at least remember doing that. 
Ok I'm going to trash this branch,  re-pull main, and start over.